site/plugins/cdn/src/Image.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Kirby\Cdn;
+
+use Kirby\Cms\App;
+use Kirby\Cms\File;
+use Kirby\Http\Url;
+use Kirby\Image\Darkroom;
+use Kirby\Image\Focus;
+
+/**
+ * Helper class for KeyCDN image processing
+ *
+ * @package   Kirby Cdn
+ * @author    Lukas Bestle <lukas@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://opensource.org/licenses/MIT
+ */
+class Image
+{
+	/**
+	 * Generates a URL with KeyCDN image processing parameters
+	 * (https://www.keycdn.com/support/image-processing)
+	 * based on a file URL or object and Kirby thumb params
+	 */
+	public static function url(string|File $file, array $params = []): string
+	{
+		$kirby = App::instance();
+
+		$root = $file;
+		if (is_object($file) === true) {
+			$root = $file->root();
+			$file = $file->mediaUrl();
+		}
+
+		$path = Url::path($file);
+
+		$darkroom = Darkroom::factory(
+			'im',
+			$kirby->option('thumbs', [])
+		);
+		$params = $darkroom->preprocess($root, $params);
+
+		$query = '';
+		if (empty($params) === false) {
+			$query = '?' . http_build_query([
+				...static::resizeOrCrop($params),
+				...static::grayscale($params),
+				...static::progressive($params),
+				...static::blur($params),
+				...static::sharpen($params),
+			]);
+		}
+
+		return $kirby->option('cdn.domain') . '/' . $path . $query;
+	}
+
+	protected static function blur(array $params): array
+	{
+		if ($params['blur'] !== false) {
+			$blur = max(0.3, min(100, $params['blur']));
+			return compact('blur');
```

Comment:
```suggestion
			return [
				'blur' => max(0.3, min(100, $params['blur']))	
			];
```
`Reasoning:` I find `compact` always a bit harder to read/understand when glancing over code, especially when quickly trying to get the difference to the other return.